framework/Source/CPTLayer.m

```diff
@@ -876,16 +875,6 @@ -(void)setNeedsDisplay
 -(void)setPosition:(CGPoint)newPosition
 {
     super.position = newPosition;
-    if ( COREPLOT_LAYER_POSITION_CHANGE_ENABLED() ) {
```

Comment:
I have no idea how this ever compiled, it must be something injected via the xcode project usually. 2.4 just removes it so 🤷